linux_os/guide/services/kerberos/kerberos_disable_no_keytab/rule.yml

```diff
@@ -0,0 +1,28 @@
+documentation_complete: true
+
+title: 'Disable Kerberos by removing host keytab'
+
+description: |-
+    Kerberos is not an approved key distribution method for
+    Common Criteria. To prevent using Kerberos by system daemons,
+    remove the host keytab <tt>/etc/krb5.keytab</tt>.
```

Comment:
The description mentions specifically `/etc/krb5.keytab` but the OVAL uses a wildcard and checks any file in `/etc` which name ends with `.keytab`.